### ISSUE TITLE
{Linked:Bug8885628}[Adaptive cards]: Labels are not associated with their respective controls.

### ISSUE THREAD

**Issue by palitsourish** [microsoft/AdaptiveCards#7650]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
URL: [Designer | Adaptive Cards](https://adaptivecards.io/designer/) (JSON Code attached)
OS Version: 22H2 (OS Build 25251.1010)
Browser: Microsoft Edge Dev Version 105.0.1300.0 (Official build) dev (64-bit) 
Screen Reader: Narrator

Repro Steps:
Turn on Narrator.
Open the above URL with JSON.
Navigate through the controls.
Observe the issue.
Actual Result:
Labels are not associated with their respective controls. Screen Reader does not announce the label associated with their respective controls.

Expected Result:
Labels should be associated with their respective controls.

Note:
Issue is also repro with NVDA.

User Experience:
Screen Reader users would be confused if Labels are not associated with their respective controls.

MAS Reference:
[Labels or Instructions - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/03.03.02)

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0)

### Screenshots

https://user-images.githubusercontent.com/95914144/177565131-f2a3102a-e896-4378-a0a5-89d771f1c9bb.mp4

### Card JSON

```JSON
{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.0",
  "body": [
    {
      "type": "ColumnSet",
      "columns": [
        {
          "type": "Column",
          "width": 2,
          "items": [
            {
              "type": "TextBlock",
              "text": "##CreateTicketFirstInputHeader##",
              "weight": "Bolder",
              "size": "Default",
              "wrap": true,
              "color": "##HeaderColor##"
            },
            {
              "type": "TextBlock",
              "text": "##CreateTicketTitleLabel##",
              "wrap": true
            },
            {
              "type": "Input.Text",
              "id": "Title",
              "placeholder": "##CreateTicketTitlePlaceholder##",
              "value": "##CreateTicketTitleValue##"
            },
            {
              "type": "TextBlock",
              "text": "##CreateTicketDescriptionLabel##",
              "wrap": true
            },
            {
              "type": "Input.Text",
              "id": "Description",
              "placeholder": "##CreateTicketDescriptionPlaceholder##",
              "value": "##CreateTicketDescriptionValue##",
              "isMultiline": true
            },
            {
              "type": "TextBlock",
              "text": "##CreateTicketCompanyCodeLabel##",
              "isVisible": "##CreateTicketCompanyCodeInputIsVisibleValue##"
            },
            {
              "type": "Input.ChoiceSet",
              "id": "CompanyCode",
              "placeholder": "##CreateTicketCompanyCodePlaceholder##",
              "value": "##CreateTicketCompanyCodeValue##",
              "choices": [ "##CompanyCodeChoices##" ],
              "isVisible": "##CreateTicketCompanyCodeInputIsVisibleValue##"
            },
            {
              "type": "TextBlock",
              "text": "##CreateTicketOrgIdLabel##",
              "isVisible": "##CreateTicketOrgIdInputIsVisibleValue##"
            },
            {
              "type": "Input.ChoiceSet",
              "id": "OrgId",
              "placeholder": "",
              "choices": [ "##OrgIdChoices##" ],
              "isVisible": "##CreateTicketOrgIdInputIsVisibleValue##"
            },

            {
              "type": "TextBlock",
              "text": "##CreateTicketCcLabel##",
              "isVisible": "##CreateTicketCcInputIsVisibleValue##"
            },
            {
              "type": "Input.Text",
              "id": "Cc",
              "placeholder": "Ex: example@microsoft.com,example2@microsoft.com ",
              "value": "##CreateTicketCCValue##",
              "isMultiline": true,
              "isVisible": "##CreateTicketCcInputIsVisibleValue##"
            },
            {
              "type": "Input.ChoiceSet",
              "id": "IsAttachment",
              "choices": [
                {
                  "title": "##CreateTicketAttachmentLabel##",
                  "value": "true"
                }
              ],
              "style": "expanded",
              "isMultiSelect": true,
              "isVisible": "##CreateTicketAttachmentIsVisibleValue##"
            }
          ]
        }
      ]
    }
  ],
  "actions": [
    {
      "type": "Action.Submit",
      "id": "SubmitButtonId",
      "title": "##ButtonProceedLabel##"
    }
  ]
}
```

**Comment by paulcam206** [comment]
The issue here is in not supplying the `label` property to `Input.Text`. See [VSO](https://microsoft.visualstudio.com/OS/_workitems/edit/40339820) bug for some more details...

**Comment by anna-dingler** [comment]
Updated JSON:

```JSON
{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.0",
  "body": [
    {
      "type": "ColumnSet",
      "columns": [
        {
          "type": "Column",
          "width": 2,
          "items": [
            {
              "type": "TextBlock",
              "text": "##CreateTicketFirstInputHeader##",
              "weight": "Bolder",
              "size": "Default",
              "wrap": true,
              "color": "##HeaderColor##"
            },
            {
              "type": "Input.Text",
              "id": "Title",
              "placeholder": "##CreateTicketTitlePlaceholder##",
              "value": "##CreateTicketTitleValue##",
              "label": "##CreateTicketTitleLabel##"
            },
            {
              "type": "Input.Text",
              "id": "Description",
              "placeholder": "##CreateTicketDescriptionPlaceholder##",
              "value": "##CreateTicketDescriptionValue##",
              "isMultiline": true,
              "label": "##CreateTicketDescriptionLabel##"
            },
            {
              "type": "Input.ChoiceSet",
              "id": "CompanyCode",
              "placeholder": "##CreateTicketCompanyCodePlaceholder##",
              "value": "##CreateTicketCompanyCodeValue##",
              "choices": [ "##CompanyCodeChoices##" ],
              "isVisible": "##CreateTicketCompanyCodeInputIsVisibleValue##",
              "label": "##CreateTicketCompanyCodeLabel##",
            },
            {
              "type": "Input.ChoiceSet",
              "id": "OrgId",
              "placeholder": "",
              "choices": [ "##OrgIdChoices##" ],
              "isVisible": "##CreateTicketOrgIdInputIsVisibleValue##",
              "label": "##CreateTicketOrgIdLabel##",
            },
            {
              "type": "Input.Text",
              "id": "Cc",
              "placeholder": "Ex: example@microsoft.com,example2@microsoft.com ",
              "value": "##CreateTicketCCValue##",
              "isMultiline": true,
              "isVisible": "##CreateTicketCcInputIsVisibleValue##",
              "label": "##CreateTicketCcLabel##",
            },
            {
              "type": "Input.ChoiceSet",
              "id": "IsAttachment",
              "choices": [
                {
                  "title": "##CreateTicketAttachmentLabel##",
                  "value": "true"
                }
              ],
              "style": "expanded",
              "isMultiSelect": true,
              "isVisible": "##CreateTicketAttachmentIsVisibleValue##"
            }
          ]
        }
      ]
    }
  ],
  "actions": [
    {
      "type": "Action.Submit",
      "id": "SubmitButtonId",
      "title": "##ButtonProceedLabel##"
    }
  ]
}
```

**Comment by vagpt** [comment]
Hi @anna-dingler, 

Thanks for sharing the JSON file.

The attached JSON file is not working at our end so could you please again share the updated file with us so that we can validate this issue at our end.

**Snippet:**
![image](https://user-images.githubusercontent.com/95913748/235159330-d899680e-607c-49a6-a19a-be12c6285033.png)
